packages/salesforcedx-vscode-core/package.nls.ja.json

```diff
@@ -26,8 +26,8 @@
   "force_config_list_text": "SFDX: すべての設定変数を一覧表示",
   "force_alias_list_text": "SFDX: すべてのエイリアスを一覧表示",
   "force_org_display_default_text": "SFDX: デフォルトの組織の詳細を表示",
-  "force_org_display_username_text": "SFDX: 組織の詳細を表示...",
-  "force_data_soql_query_input_text": "SFDX: SOQL クエリを実行...",
+  "force_org_display_username_text": "SFDX: 組織の詳細を表示",
```

Comment:
Let's not touch the other commands for now

Reply:
Done!!!